src/storage/src/sink/healthcheck.rs

```diff
@@ -268,19 +300,25 @@ mod tests {
         tokio::time::advance(Duration::from_millis(1)).await;
         let error = String::from("some error here");
         healthchecker
-            .update_status(SinkStatus::Failed(error.clone()))
+            .update_status(SinkStatus::Failed {
+                error: error.clone(),
+                hint: None,
```

Comment:
why not test with a hint here?

Reply:
Oh I totally ignored the actual code here as part of the method refactor. Test with a hint totally makes sense here, will add 👍

Reply:
Added!